src/pages/home/report/ReportActionCompose.js

```diff
@@ -220,6 +219,11 @@ class ReportActionCompose extends React.Component {
     setTextInputRef(el) {
         ReportActionComposeFocusManager.composerRef.current = el;
         this.textInput = el;
+
+        // I had an issue here when I set up reportActions in another component,
+        // I believe the loading delay prevented the element
+        // from being set here before compoenentDidMount was called
+        this.updateComment(this.comment);
```

Comment:
Can you elaborate more descriptively, please? With more details if possible before/now behaviour videos too